### DISCUSSION TITLE
SMTP/email debugging

### DISCUSSION THREAD

**Discussion by dadcoachengineer** [mealie-recipes/mealie#1647] [Q&A]
Is there an env to toggle on additional detail for SMTP/email subsystem?

INFO: 13-Sep-22 16:36:06 	send email result: <emails.backend.SMTPResponse status_code=None status_text=None>
ERROR: 13-Sep-22 16:36:06 	send email error: [Errno None] Connection unexpectedly closed: timed out: None

This is not terribly useful information. My variables are set to known working values but my syntax may be wrong.

Thanks!

**Comment by ramsnerm** [comment]
Did you get it to work? I am also stuck in the mail setup topics.
